packages/charts/src/charts/bar/bar-chart.component.tsx

```diff
@@ -205,6 +210,7 @@ export const BarChart: FC<Props> = ({
                   selectors={tooltip.selectors}
                   isList={tooltip.selectors.length > 1}
                   formatValue={formatTooltip}
```

Comment:
Use `TooltipFormatter` from `@keen.io/charts-utils` and format value with `formatValue()`.